src/entity/src/user.js

```diff
@@ -889,13 +889,25 @@ export default class User {
   }
 
   /**
+   * Find users.
+   *
+   * @param {Query} [query] Query used to filter entities.
+   * @param {Object} [options] Options
+   * @param {boolean} [options.discover] Discover users.
+   * @return {Observable} Observable.
+   */
+  static find(query, options = {}) {
```

Comment:
Same comment as about `find` vs `lookup`. Also a side note - it's probably easier for us to be prescriptive about the methods that devs should use - i.e. we either expose this on `UserStore` or `User`. Maintaining both seems redundant and unnecessary overhead.